services/mailer/index.ts

```diff
@@ -17,9 +17,8 @@ const app = express()
 const validateRequest = (req, res, next) => {
     const isInvalidated: boolean = !req.headers['x-hasura-role'] || req.headers['x-hasura-role'] === 'anonymous'
     const isIntrospection: boolean = req.body['operationName'] === 'IntrospectionQuery'
-    const isInternal: boolean = ['sendMail'].some(service => req.body['query'].includes(service))
 
-    if (isInvalidated && isInternal && !isIntrospection) {
+    if (isInvalidated && !isIntrospection) {
```

Comment:
Leave it as internal for now, it's related to future planning. Also, irrelavant to the ticket.